src/classes/STG_InstallScript.cls

```diff
@@ -74,8 +74,8 @@ global without sharing class STG_InstallScript implements InstallHandler {
             populateAdminRecordTypeSetting(previousVersion);
 
             setDefaultContactLanguageFluency(previousVersion);
-
-            enablePreventDeletionHierarchyCustomSettings(previousVersion); 
+            
```

Comment:
shall we remove these indentations?

Reply:
Also, there is some other funky indentations in this class that i noticed. Could we clean them up using this chance?

Reply:
did some minor changes in the class, let me know if this looks good.